tests/dependency_analyzer/cyclonedx/test_cyclonedx.py

```diff
@@ -106,3 +107,11 @@ def test_multiple_versions(snapshot: dict[str, DependencyInfo]) -> None:
     bom_path = Path(RESOURCES_DIR, "bom_multi_versions.json")
     result = get_deps_from_sbom(bom_path)
     assert snapshot == result
+
+
+def test_custom_sbom_name_with_maven() -> None:
+    """Test reading cyclonedx maven sbom that was created using a custom name."""
+    cyclonedx: CycloneDxMaven = CycloneDxMaven(
+        "", "bom.json", "maven", defaults.get("dependency.resolver", "dep_tool_maven"), "localhost"
+    )
+    assert cyclonedx.collect_dependencies(str(RESOURCES_DIR))
```

Comment:
Should we compare the actual value of the returned dictionary instead of checking whether the dictionary is not empty?

Reply:
We are really only testing that the plugin can find the json file when it has a custom name. Verifying the contents is outside of the scope I'd say.